prow/prowjobs/kubeflow/pipelines/kubeflow-pipelines-presubmits.yaml

```diff
@@ -171,6 +171,16 @@ presubmits:
         command:
         - ./test/presubmit-components-gcp.sh
 
+  - name: kubeflow-pipelines-components-google-cloud-python38
+    cluster: build-kubeflow
+    decorate: true
+    run_if_changed: "^(components/google-cloud/.*)|(/pipelines/components/google-cloud/tests/presubmit.sh)$"
+    spec:
+      containers:
+      - image: python:3.8
+        command:
+        - ./pipelines/components/google-cloud/tests/presubmit.sh
```

Comment:
You can write

./components/google-cloud/tests/presubmit.sh

Current working dir is repo root